pennylane/ops/op_math/controlled.py

```diff
@@ -470,6 +471,8 @@ def has_decomposition(self):
             return True
         if isinstance(self.base, qml.PauliX):
             return True
```

Comment:
Yes, because with the ZYZ decomposition, the decomposition will have 6 gates, two of which will be `MultiControlledX`. The ZYZ decomposition only disregards the rotations that are trivial, but 4 of the 6 rotations for the controlled ZYZ decomposition of `PauliX` are non-trivial.